.github/workflows/buf-pr.yaml

```
@@ -0,0 +1,31 @@
+name: Buf PR
+on:
+  pull_request:
+    types: [opened,synchronize,reopened,labeled,unlabeled]
+permissions: read-all
+jobs:
+  buf-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Buf setup
+        uses: bufbuild/buf-setup-action@v1
+        with:
+          version: 1.32.0-beta.1
+      - run: buf --version
+      - name: Buf build
+        run: buf build . --error-format github-actions
```

I don't think the dot is necessary? I would clean this up everywhere
```suggestion
        run: buf build --error-format github-actions
```